beacon-chain/sync/rpc.go

```diff
@@ -88,3 +96,35 @@ func (r *RegularSync) registerRPC(topic string, base interface{}, handle rpcHand
 		}
 	})
 }
+
+func (r *RegularSync) chunkedHandler(stream libp2pcore.Stream, msg interface{}) error {
```

Comment:
Please add a comment to this method. Is it supposed to read all chunks on the stream until EOF? Or does it read a single chunk message (i.e. reads a single block).

Please a choose a more appropriate method name. "Handler" is too vague. This method writes a single chunk to the stream, but it's not clear without reading through the implementation.